packages/jest-runner/src/runTest.ts

```diff
@@ -142,6 +142,13 @@ async function runTestInternal(
     docblockPragmas,
     testPath: path,
   });
+
+  if (typeof environment.getVmContext !== 'function') {
+    throw new Error(
+      `Test environment found at "testEnvironment" does not export a "getVmContext" method, which is mandatory from Jest 27`,
```

Comment:
I think it would be nice to mention that it's a replacement for `runScript`
```suggestion
      `Test environment found at "testEnvironment" does not export a "getVmContext" method, which is mandatory from Jest 27. This method is a replacement for "runScript".`,
```

Reply:
good idea 👍